packages/shared/src/role.types.ts

```diff
@@ -14,8 +14,19 @@ export type RoleSerialized = {
   canManageGroup: boolean;
   canManageRoster: boolean;
   canViewRoster: boolean;
+  canViewObservation: boolean;
```

Comment:
Is this permission being used to hide the observation page if they do not have it?